src/NITTA/LuaFrontend.hs

```diff
@@ -392,8 +392,13 @@ alg2graph LuaAlgBuilder{algGraph, algLatestLuaValueInstance, algVars} = flip exe
 lua2functions src =
     let syntaxTree = getLuaBlockFromSources src
         luaAlgBuilder = buildAlg syntaxTree
-        frTrace = getFrTrace $ algTraceFuncs luaAlgBuilder
+        frTrace = getFrTrace $ getAllTraceFuncs luaAlgBuilder
      in FrontendResult{frDataFlow = alg2graph luaAlgBuilder, frTrace = frTrace, frPrettyLog = prettyLog frTrace}
+    where
+        getAllTraceFuncs algBuilder =
+            let traceFuncs = algTraceFuncs algBuilder
+                startupArgNames = map (fst . snd) $ HM.toList $ algStartupArgs algBuilder
```

Comment:
Here, explicit pattern matching with variable naming will be better than `fst . snd`